identity/src/lib/rpc/mod.rs

```diff
@@ -1,2 +1,47 @@
 pub mod server;
 pub mod service;
+
+use std::{io, net::SocketAddr, sync::Arc};
+
+use futures::{future, prelude::*};
+use tarpc::client::Config;
+use tarpc::server::{BaseChannel, Channel, Handler};
+use tokio_serde::formats::Json;
+
+use self::server::IdentityServer;
+use self::service::{IdentityService, IdentityServiceClient};
+use crate::{config::Configuration, db::Db};
+
+pub async fn rpc_server(
+    addr: SocketAddr,
+    configuration: Arc<Configuration>,
+    db: Arc<Db>,
```

Comment:
Can you please check if the `Arc<...>` is really needed? If not please refactor it in a next separate PR :)